app/views/genres/index.html.erb

```diff
@@ -0,0 +1,11 @@
+<% @genres.each do |genre| %>
+  <%= link_to genre.name, genre_path(genre) %>
+<% end %>
+
+<% if current_user && current_user.admin? %>
+  <%= form_for @genre do |f| %>
+    <%= f.label :name %>
+    <%= f.text_field :name %>
+    <%= f.submit 'Submit' %>
```

Comment:
better practice is to let Ruby do its magic and populate the button text with "Create Genre" or "Edit Genre" if you reuse the form etc.
![image](https://media2.giphy.com/media/12NUbkX6p4xOO4/giphy.gif)